judge/tasks/webhook.py

```diff
@@ -130,17 +129,43 @@ def on_new_tag(problem_code, tag_list):
     for tag in tag_list:
         tags.append(Tag.objects.get(code=tag).name)
 
-    url = site_url + problem.get_absolute_url()
+    url = settings.SITE_FULL_URL + problem.get_absolute_url()
 
     description = f'Title: {problem.name}\n'
     description += f'New tag: {", ".join(tags)}'
 
-    webhook = DiscordWebhook(url=webhook)
-    embed = DiscordEmbed(
-        title=f'New tag added for problem {url}',
-        description=description,
-        color='03b2f8',
-    )
+    send_webhook(webhook, f'New tag added for problem {url}', description, None)
 
-    webhook.add_embed(embed)
-    webhook.execute()
+
+@shared_task
+def on_new_contest(contest_key, is_suggested=False):
```

Comment:
Delete `is_suggested`

Reply:
w..w..w..w..w..why

Reply:
It looks like a leftover from copying `on_new_problem`. You also don't use it in the code.

Reply:
oh you r right, my bad

Reply:
Fixed in new commit